### PR TITLE
disable blank issues

### PR DESCRIPTION
**Pull request by niraj8**
A contributor/reporter might create a blank issue, the change here disables this behaviour

### CODE REVIEW

**Comment by niraj8** [comment]
The change here won't actually fix the linked issues😅
But I think it would be okay to disable the blank issues anyway

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
```
Comment (thooria) [inline .github/ISSUE_TEMPLATE/config.yml:1]: false

**Comment by thooria** [comment]
thoori

**Comment by thooria** [comment]
B6358C87-148D-4D29-A241-FB51E46E2548
#

**Pull request closed without merging**